Make some minor cleanup and corrections to the DartPad page (#2975)
dart-lang/site-www: src/tools/dartpad/index.md

DartPad supports `dart:*` [core libraries](/guides/libraries) marked as
multi-platform and web platform. It doesn't support those marked native

> platform nor does it support [deferred loading][].

Reviewer: ```suggestion
platform, and it doesn't support [deferred loading][].
```

  Go to <a href="{{site.dartpad}}" target="_blank" rel="noopener">DartPad.</a>

>   A sample appears on the left and the output appears on the right.

Reviewer: I think auto-run if off for Dart samples, to avoid problems with endless loops. Also, if the reader has played with DartPad before, they might not see a sample. Maybe:

```suggestion
  Dart code appears on the left, and
  a place for the output appears on the right.
```

  </li>

  <li markdown="1">
  Choose a Flutter sample like **Sunflower**,

>   using the **Samples** list in the upper right.

Author: I suppose "at" is more semantically correct in this case, I'll switch it back :)

>   using the **Samples** list in the upper right.

Reviewer: I was honestly curious. Prepositions are weird things.

>   Again, the rendered output appears to the right.

Reviewer: ```suggestion
  The rendered output appears to the right.
```

  </li>

>   <li markdown="1">
>   Click **CONSOLE** to view the sample's console output.
>   </li>

Reviewer: Maybe we should delete this? The Sunflower app has no console output.

```suggestion
```

  Click the **New Pad** button,
  and confirm that you want to discard changes to the current pad.

>   Click on the Dart logo to access the source code for
>   the Hello World app.

Reviewer: Let's move this down to the next step

```suggestion
```

  </li>

  <li markdown="1">

>   Using the toggle underneath the logo,
>   verify **HTML** support is disabled.

Reviewer: ```suggestion
  Click the Dart logo, make sure that **HTML** support is disabled,
  and then click **Create**.
```

Btw, we don't say "click _on_" (https://developers.google.com/style/word-list#letter-c), and we try to be more specific than _access_ (https://developers.google.com/style/word-list#letter-a).
